action/candidate_register.go

```diff
@@ -103,6 +103,7 @@ type CandidateRegister struct {
 	duration        uint32
 	autoStake       bool
 	payload         []byte
+	candidateType   uint32
```

Comment:
would this be an issue for existing candidate-related funcs?

Reply:
for example, consensus delegate is qualified when amount >= 1.2M, while ENode is qualified when amount >= 10k